packages/swapper/src/api.ts

```diff
@@ -256,13 +284,11 @@ export interface Swapper<T extends ChainId> {
   /**
    * Get builds a trade with definitive rate & txData that can be executed with executeTrade
    **/
-  buildTrade(args: BuildTradeInput): Promise<Trade<T>>
-
+  buildTrade(args: BuildTradeInput): Promise<Result<Trade<T>, SwapErrorMonad>>
```

Comment:
e.g. here reads more correct as `SwapErrorLeft`
```suggestion
  buildTrade(args: BuildTradeInput): Promise<Result<Trade<T>, SwapErrorLeft>>
```